test/setup/wdio.conf.ts

```diff
@@ -0,0 +1,141 @@
+/* eslint camelcase: ["error", {properties: "never"}] */
+/**
+ * See also: http://webdriver.io/guide/testrunner/configurationfile.html
+ */
+
+import { Options } from '@wdio/types';
+import { existsSync } from 'fs';
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+import type { Capabilities } from '@wdio/types';
+import JsonReporter from '../helpers/json-reporter.js';
+import TestEnv from './TestEnv.js';
+
+// eslint-disable-next-line no-underscore-dangle
+const __dirname = dirname( fileURLToPath( import.meta.url ) );
+
+export function wdioConfig( providedTestEnv: TestEnv ): WebdriverIO.Config {
+	global.testEnv = providedTestEnv;
+	const settings = testEnv.settings;
+
+	return {
+		specs: settings.specs.map( ( specFilepath ) => `${__dirname}/../${specFilepath}` ),
+
+		hostname: 'browser',
+		port: 4444,
+		path: '/wd/hub',
+		// ============
+		// Capabilities
+		// ============
+		// https://sites.google.com/a/chromium.org/chromedriver/capabilities
+		capabilities: [
+			{
+				browserName: 'chrome',
+				maxInstances: settings.maxInstances,
+				'goog:chromeOptions': {
+					args: [
+						// The window size is relevant for responsive pages rendering differently on
+						// different screen sizes. Bootstrap considers widths between 1200 and 1400
+						// as XL, let's use that.
+						// https://getbootstrap.com/docs/5.0/layout/breakpoints/#available-breakpoints
+						...[ '--window-size=1280,800' ],
```

Comment:
Please just use `'--window-size=1280,800'` instead of `...[ '--window-size=1280,800' ]`

Reply:
For sure, I can and agree, but this is current code, nothing I changed.